APIs/openEO/fair.qmd

```diff
@@ -9,23 +9,26 @@ aliases:
 
 ## FAIR
 
-One of the key goals of openEO, is to support [FAIR principles](https://www.go-fair.org/fair-principles/) and open science. The implementation in the Copernicus dataspace
-makes it easier to comply with these principles, by incorporating these principles in the implementation, so that
-users are automatically a step closer to generating FAIR-compliant open data. If your project has requirements related to these topics,
+One of the key goals of openEO, is to support [FAIR principles](https://www.go-fair.org/fair-principles/) and open science. Directly implementing these principles in Copernicus Dataspace makes it easier to follow them. In other words,  with the use of openEO, users are creating FAIR-compliant open solutions automatically. If your project has requirements related to these topics,
 this should serve as a good starting point.
 
 These are a few examples:
 
 - *[F2 Rich metadata](https://www.go-fair.org/fair-principles/f2-data-described-rich-metadata/)* openEO generates rich STAC metadata,
   that includes processing info, complete raster metadata, band information, etcetera.
-- *[R1.2 Detailed provenance](https://www.go-fair.org/fair-principles/r1-2-metadata-associated-detailed-provenance/)* In result metadata [derived-from](https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md#derived_from)
-  links link back to all input products to provide provenance.
+- *[R1.2 Detailed provenance](https://www.go-fair.org/fair-principles/r1-2-metadata-associated-detailed-provenance/)* As a result, metadata "[derived-from](https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md#derived_from)" links trace back to all input products, ensuring provenance. 
```

Comment:
I was referring here to 'result metadata', so the metadata associated with a result generated by openEO.